### ISSUE TITLE
Hosting/Server

### ISSUE THREAD

**Issue by ILoveBrains** [toomanybooks/grubs-tavern#4]
Are we going to be using something like AWS or am I still going to fabricoble something together 
If we could come up with some Min specs we would need for everything so that I can make sure I've got something that works and doesn't eat too much electricity that would be great.

**Comment by toomanybooks** [comment]
Good point....it might be best to host it on AWS or similar IaaS provider to see what we minimally need. a couple benefits of AWS (only one I've used) is that we will probably be within the free-tier usage and it's real fast and easy.
